### ISSUE TITLE
Adding aria-labels, ids to public forms for accessibility

### ISSUE THREAD

**Issue by candideu** [knadh/listmonk#697]
# **Is your feature request related to a problem? Please describe.**
Hello! I've been really enjoying the latest release, especially the ability to edit the css and js of the public pages and admin panel.

That said, I noticed that the public subscription forms are missing key attributes like #ids, `for=`, and aria-labels. These are helpful for accessibility, or to further customize things like placeholders (see: [Labeling Controls](https://www.w3.org/WAI/tutorials/forms/labels/)).

![image](https://user-images.githubusercontent.com/55474996/153737291-5e005b50-40c1-4ad1-b3e5-1083fcd3b361.png)

# **Describe the solution you'd like**
Code at the moment:

```html
  <p>
    <label>E-mail</label>
    <input name="email" required="true" type="email" placeholder="E-mail" autofocus="true">
    <input name="nonce" class="nonce" value="">
  </p>
  <p>
    <label>Name (optional)</label>
    <input name="name" type="text" placeholder="Name (optional)">
  </p>
```

I propose something like this:

```html
  <p>
    <label for="signup-form-email">E-mail</label>
    <input name="email" required="true" type="email" placeholder="E-mail" autofocus="true" id="signup-form-email" aria-label="email">
    <input name="nonce" class="nonce" value="">
  </p>
  <p>
    <label for="signup-form-name">Name (optional)</label>
    <input name="name" type="text" placeholder="Name (optional)" id="signup-form-name" aria-label="name">
  </p>
```

**Comment by candideu** [comment]
Thanks for the fix @shrilakshmishastry
